docs/sphinx/sections/api/apidocs/internals.rst

```diff
@@ -214,8 +214,8 @@ Exception handling
 
 ----
 
-Step Launchers (Experimental)
------------------------------
+Step Launchers (Superseded)
+-------------------------------------------------------------------------------------------------
```

Comment:
I believe Sphinx will throw a warning that the underling is not the same length as the title.